src/gnn_reco/components/loss_functions.py

```diff
@@ -0,0 +1,30 @@
+import torch
+import numpy as np
+from torch.nn import CrossEntropyLoss
+
+
+def LogCosh(prediction, graph, target):
```

Comment:
As this is used as a function (compared to e.g. `CrossEntropyLoss` from `torch.nn`, which is a class) I would propose using "snake_case" here, as well as in L9 and L14 of this file. Alternatively, these three could be changed to classes to conform with torch standards. I would probably vote for the latter, but have no strong opinions as long as we're just following standard naming conventions.

Reply:
I changed the names